src/tile.ts

```diff
@@ -1,14 +1,26 @@
-import { interaction, Point, Graphics } from 'pixi.js'
+import { interaction, Point, Graphics, Container } from 'pixi.js'
 import makeTileModel from './tile-model-factory'
 import TileGraphics from './tile-graphics'
 
+type Coords = { x: number; y: number }
+
+export type InteractionHandler = (e: interaction.InteractionEvent, context: InteractionContext) => void
+
+export type InteractionContext = {
+  sprite: Graphics
+  startContainer?: Container
+  startPosition?: Coords
+}
+
 class Tile {
   sprite: Graphics
   private interactionData: interaction.InteractionData | null = null
   private deltaDragPoint = new Point(0, 0)
   private isDragging: boolean = false
+  private startPosition?: Coords
+  private startContainer?: Container
 
-  constructor() {
+  constructor(private stage: Container) {
```

Comment:
Doesn't look like you're doing anything with this yet, but does it make sense for a child to know things about its parent?

Reply:
There likely is a better way to do this, but right now I am using stage [here](https://github.com/drewwyatt/railroad/pull/6/files#diff-fa3eec171b73cf555bc76d7516a623faR69) to pop the tile out of it's current container while it is being dragged